app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const Greeting = () => {
+  const { greeting, isLoading, isError, errorMsg } = useSelector((store) => store.greeting);
+
+  console.log(greeting.message);
+
+  if (isLoading) {
+    return (<h1>Loading Greeting</h1>);
+  }
+
+  if (isError) {
+    return (<h1>{ errorMsg }</h1>);
+  }
```

Comment:
Currently, your application keeps rendering an error message even after populating the database with the respective salutations😉. Please let's try to fix this👍🏼.
<img width="1438" alt="image" src="https://github.com/badger-99/hello-rails-react/assets/64643995/b5326d72-0c6f-4f87-9d5f-bebb7c0a3ffa">